package.json

```diff
@@ -31,9 +31,12 @@
     "debug": "export SLS_DEBUG=* && node --inspect ./node_modules/serverless/bin/serverless offline -s dev --noAuth --config serverless-api.yml",
     "start-jigsaw-docs": "sls offline start -s dev --port 3015 --noAuth --config serverless-jigsaw-docs.yml",
     "auth": "serverless invoke local -s dev -f hn-single-view-api-authorizer -p authorizer/event.json --config serverless-api.yml",
+    "debug-test": "export SLS_DEBUG=* && node --inspect ./node_modules/serverless/bin/serverless offline -s test --port 3010 --noAuth",
     "unit-tests": "jest --testPathIgnorePatterns=api/test/integration --coverage",
-    "start-test": "sls offline start -s test --port 3010 --noAuth --config serverless-api.yml",
+    "start-test": "sls offline start -s test --port 3010 --noAuth --config serverless-api.yml & node ./api/test/FakeJigsaw.js",
     "integration-tests": "jest --testPathPattern=api/test/integration",
+    "integration-tests-local": "jest --testPathPattern=api/test/integration --watch",
+    "ci-local": "start-server-and-test start-test http://localhost:3010/customers?firstName=jon integration-tests-local",
```

Comment:
can we take these locals out? npm run ci is also local, all the new ones do is add a "--watch" and the scripts are cluttered already. you can add --watch while developing